elasticsearch/prep-install.sh.origin

```diff
@@ -0,0 +1,6 @@
+ln -s /usr/share/elasticsearch /usr/share/java/elasticsearch 
```

Comment:
yeah, dunno if it's best practice to have it there or not.  I always put it on but the man page says it executes in the current shell anyway so I guess it's not a hard requirement